tests/lib/rules/no-node-access.test.ts

```diff
@@ -51,22 +51,38 @@ ruleTester.run(RULE_NAME, rule, {
         within(signinModal).getByPlaceholderText('Username');
       `,
     },
-    {
+    /*{
```

Comment:
I'll reenable this test in the PR part 3 since there is where I'm gonna include the test file name pattern to report or not rules, so this will be solved then.